vmdb/app/helpers/container_service_helper/textual_summary.rb

```diff
@@ -7,16 +7,18 @@ def textual_group_properties
     items = %w(
       namespace
       name
-      port
       creation_timestamp
       resource_version
       session_affinity
-      portal_ip
-      protocol
-      container_port)
+      portal_ip)
     items.collect { |m| send("textual_#{m}") }.flatten.compact
   end
 
+  def textual_port_configs
```

Comment:
@Fryguy 
![gui-example](https://cloud.githubusercontent.com/assets/11256940/7180008/109cc072-e446-11e4-8c82-c24a58bb2b3f.png)
[Here](https://github.com/ManageIQ/manageiq/pull/2523/files#r28372557) is where this method is called and port configurations are displayed.